kimchi/src/verifier.rs

```diff
@@ -610,20 +613,19 @@ where
                             panic!("Attempted to use {:?}, but no lookup index was given", col)
                         }
                         Some(lindex) => {
+                            let joint_combiner =
+                                constants.joint_combiner.expect("missing joint combiner");
```

Comment:
It is only set to `None` if we didn't follow the flow correctly. It already defaults to `Some(zero)` in the case were we only do single column lookups.

<img width="624" alt="Screen Shot 2022-05-03 at 8 35 51 AM" src="https://user-images.githubusercontent.com/1316043/166413104-c241ec60-dfc5-4939-8255-9e8695dc7155.png">
 
PS: I also believe we can't, today, do a single column lookup because `SingleLookup` doesn't have a `table_id` entry :D